src/WebJobs.Script.WebHost/WebScriptHostBuilderExtension.cs

```diff
@@ -42,7 +45,7 @@ public static class WebScriptHostBuilderExtension
 
                     configureWebJobs?.Invoke(webJobsBuilder);
 
-                    ConfigureRegisteredBuilders(webJobsBuilder, rootServiceProvider);
+                    configureWebJobsBuilder = () => ConfigureRegisteredBuilders(webJobsBuilder, rootServiceProvider);
```

Comment:
@fabiocav / @mathewc 
Well, my plan to move all the services up failed spectacularly. We register a bunch of stuff *after* we call the configureWebJobs callback inside `AddScriptHost`. In order to keep things clean (from the test perspective), I changed this to capture that ConfigureWebJobs call and move it to the very end... this now works and all tests are passing except one flaky one that Pragna is fixing (it's unrelated).

Reply:
Wondering if your other approach, though not as clean, would avoid subtle issues that may arise from capturing the builder to use at a later stage.

Reply:
It does feel safer. I'll do that and then commit this so we can unblock some other things.